src/dev/flang/ast/AstErrors.java

```diff
@@ -1579,15 +1579,23 @@ static void lossOfPrecision(SourcePosition pos, String _originalString, int _bas
       "Expected number given in base " + _base + " to fit into " + _type + " without loss of precision.");
   }
 
-  public static void argumentNamesNotDistinct(SourcePosition pos, Set<String> duplicateNames)
+  public static void argumentNamesNotDistinct(Feature f, Set<String> duplicateNames)
   {
-    error(pos,
-      "Names of arguments used in this feature must be distinct.",
+    int[] cnt = new int[1];
+    error(f.pos(),
+          "Names of arguments used in this feature must be distinct.",
           "The duplicate" + (duplicateNames.size() > 1 ? " names are " : " name is ")
           + duplicateNames
             .stream()
             .map(n -> sbn(n))
             .collect(Collectors.joining(", ")) + "\n"
+          + "Feature with equally named arguments: "+ s(f) + "\n"
```

Comment:
Somehow this sounds a bit strange to me. Maybe "Feature contains duplicate argument names"?

Reply:
Not sure, but sounds alright to me. We need a native speaker to proof read everything at some point...